### ISSUE TITLE
ArCore 1.8. no more running on P20 Pro (ArCore 1.7 is ok)

### ISSUE THREAD

**Issue by MWE-Realnote** [google-ar/arcore-android-sdk#758]
Since ArCore 1.8 ArCore is no longer running on Huawei P20 Pro.

Nearly all the time  `frame.getCamera().getTrackingState()` is TrackingState.PAUSED,
with `getTrackingFailureReason()` == "NONE".

This true for a secound Apps.
Our Apps with the identical APK is running on Motorola G5S.

Our App ist running correct on P20 Pro wirh Arcor 1.7.

**Comment by KirillRib** [comment]
Hi, I am facing the same problem with my Huawei P20 Pro. All applications using ArCore stopped working after last ArCore update.

**Comment by KirillRib** [comment]
> Our App ist running correct on P20 Pro wirh Arcor 1.7.

How did you manage to launch the application using Arcor 1.7? When I try to start the application using Arcor 1.7, the application requires updating Arcor.

**Comment by antoan-boss** [comment]
Same issue here with P20 Pro...

**Comment by MWE-Realnote** [comment]
I downgraded the ArCore Version in our App, and removed / deinstalled  ArCore.
Then I installed an older APK with 1.7., and the App is again running fine.

**Comment by brookman** [comment]
We have the same problem.

**Comment by fredsa** [comment]
Anyone seeing an improvement with ARCore 1.9 on this device? Or still seeing the same issues?

Also, it would be helpful to provide the following information:

### VERSIONS USED
- Android Studio: 
- ARCore SDK for Android: 
- Device manufacturer, model, and O/S: 
- ARCore: 
  On Windows, use: `adb shell pm dump com.google.ar.core | findstr /i "packages: versionName"`
  On macOS, use: `adb shell pm dump com.google.ar.core | egrep -i versionName\|packages:`
- Output of `adb shell getprop ro.build.fingerprint`: 


### STEPS TO REPRODUCE THE ISSUE
 1.
 1.
 1.

**Comment by AJoensuu** [comment]
@fredsa v1.8 did not work, but v1.9 works.
All good here. Thanks.

**Comment by brookman** [comment]
We have a Huawei P20 Pro test device here and no ARCore apps are running on it. I have tried the official Measure app from the Play Store.
Error:

```
2019-07-30 12:37:48.075 32435-32446/? E/native: calibration_jni.cc:189 No device profile available for build fingerprint HUAWEI/CLT-L29/HWCLT_accelerometer-icm20690_-1:8.1.0/HUAWEICLT-L29/152(C432):user/release-keys
2019-07-30 12:37:48.079 32176-32243/ch.sbb.ar.dev E/native: session_create_implementation_shared.cc:725 Unable to obtain device profile from the content provider: generic::not_found: Device is not supported.
2019-07-30 12:37:48.079 32176-32243/ch.sbb.ar.dev E/native: session_create_implementation_shared.cc:1680 Failed to create calibration provider.
```

Model: CLT-L29
Build: CLT-l29 8.1.0.152(C432) GPU Turbo (System says this is the newest version)
EMUI version: 8.1.0
Android version: 8.1.0

ARCore version: 1.10.190603116

**Comment by sandyfriend** [comment]
it should be fixed on 1.11 release. please re-open it problem exists.
